stm32_part_table.yaml

```
@@ -747,3 +747,13 @@ stm32wb:
     rm_url: https://www.st.com/resource/en/reference_manual/dm00318631-multiprotocol-wireless-32bit-mcu-armbased-cortexm4-with-fpu-bluetooth-lowenergy-and-802154-radio-solution-stmicroelectronics.pdf
     members:
       - STM32WB55
+
+stm32u5:
+  stm32u5x5:
+    url: https://www.st.com/en/microcontrollers-microprocessors/stm32u5-series.html
+    rm: RM0456
+    rm_title: STM32U575 and STM32U585
+    rm_url: https://www.st.com/resource/en/reference_manual/rm0456-stm32u575585-armbased-32bit-mcus-stmicroelectronics.pdf
+    members:
+      - STM32U575
+      - STM32U575
```

```suggestion
      - STM32U575
      - STM32U585
```

Maybe typo?  Might be related to only having a `stm32u5` feature.

---

Thank you for spotting this 👍🏾